sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -240,3 +240,31 @@ export class Application extends RecordDataModel {
   )
   disbursementSchedules?: DisbursementSchedule[];
 }
+
+/**
+ * Interface for assessment payload
+ */
+export interface Assessment {
+  weeks: number;
+  totalFederalAward: string;
```

Comment:
Can you elaborate on the reason to keep all these values as strings?
Are they saved as strings to the assessment payload on _sims.applications_ table?

Reply:
Type of the assessment properties changed to number and Math.round used as discussed.